extra/CMakeLists.txt

```diff
@@ -74,6 +72,15 @@ else()
 	set(WITH_CPLEX_INCDIR "")
 endif()
 
+if(GUROBILIB)
+	set(GUROBI_LIB "-L${GUROBI_LIB_DIR} -lgurobi${GUROBI_VERSION} -lm -lpthread")
```

Comment:
The library name should be captured by cmake variable. Try `GUROBILIB` or find the relevant cmake manual.